src/Text/Pandoc/Writers/OpenDocument.hs

```diff
@@ -255,7 +255,7 @@ writeOpenDocument opts (Pandoc meta blocks) = do
                collectBlockIdent _                             = []
            modify $ \s -> s{ stIdentTypes = query collectBlockIdent blocks }
            m <- metaToContext opts
-                  (blocksToOpenDocument opts)
+                  (inlinesToOpenDocument opts . B.toList . blocksToInlinesWithSep (B.str "<text:line-break/>"))
```

Comment:
This doesn't seem quite right -- have you tested with multiparagraph content?
I think what you want is not `B.str "<text:linebreak/>"` but `B.rawInline (Format "opendocument") "<text:linebreak/>"`.

Reply:
oh, i didn't notice that this exists.
another thing i didn't notice is B.lineBreak which is easier.
another thing i didn't notice is "blocksToInlines" uses B.lineBreak by default.

hmm, for the input format used here multiple paragraphs in title result in a SoftBreak and not Para elements, so i'll just use blocksToLines.

btw i had to rebase this on the other (new) PR due to the version number change, which affects the test result - the top commit is the only one that "belongs" to this PR.